tests/base_test.py

```diff
@@ -55,6 +55,14 @@
 
 
 class ArchesTestRunner(DiscoverRunner):
+    def __init__(self, *args, **kwargs) -> None:
+        kwargs["debug_mode"] = True
+        # Unless the user has something other than the Django default, give them
+        # what they probably want.
+        if kwargs["pattern"] == "test*.py":
+            kwargs["pattern"] = "*.py"
```

Comment:
Since I was already patched in here to turn on `--debug-mode`, figured I'd set a better default for `--pattern` so that `manage.py test` works more like it used to before 7.5.